apiary.apib

```diff
@@ -371,6 +371,7 @@ Get a list of contacts.
             + ids: `cb8da52a-ce89-4bf6-8f7e-8ee6cb85e3b5`,`f8a57a6f-dd1e-41a3-b8d3-428663f1d09e` (array[string], optional)
             + term: `James` (string, optional) - Filters on first_name, last_name, email and telephone
             + updated_after: `2016-02-04T16:44:33+00:00` (string, optional)
+            + tags: `expo`,`prospect` (array[string], optional) - Filters on contacts coupled to all given tags
```

Comment:
Does this lint? I think in other examples we dropped the quotes `expo, prospect`

Reply:
Yep, I've tested this manually and this is displayed correctly.